### PR TITLE
feat(js): Add documentation for new `ContextLines` integration

### PR DESCRIPTION
**Pull request by Lms24**
Adds documentation for the new `ContextLines` integration which we're going to ship soon (https://github.com/getsentry/sentry-javascript/pull/8699)

ref https://github.com/getsentry/sentry-javascript/issues/8656

### CODE REVIEW

**Comment by Lms24** [comment]
Closing for the time being as I had to revert the PR in the SDK. We might drop the feature. I'll reopen if we decide to proceed.

**Review by lforst — APPROVED** [review]

File: src/includes/platforms/configuration/integrations/plugin.mdx [hunk 1]
```diff
@@ -107,3 +107,20 @@ Available options:
...
 
 </PlatformSection>
+
+<PlatformSection supported={["javascript"]}>
+
+### ContextLines
+
+_Import name: `Sentry.Integrations.ReportingObserver`_
+
+This integration collects code lines at and around the stack frames of an error that point to code in the current page's html (for example in `<script>` tags).
```
[the remark this answers is not in the document]
Comment (Lms24) [reply src/includes/platforms/configuration/integrations/plugin.mdx]: much better! thx for the suggestion

File: src/includes/platforms/configuration/integrations/plugin.mdx [hunk 2]
```diff
@@ -107,3 +107,21 @@ Available options:
...
 
 </PlatformSection>
+
+<PlatformSection supported={["javascript"]}>
+
+### ContextLines
+
+_Import name: `Sentry.Integrations.ContextLines`_
+
+This integration attaches source code of inline JavaScript inside the page's HTML (e.g. JS in `<script>` tags) to errors captured by the SDK.
```
Comment (shanamatthews) [inline src/includes/platforms/configuration/integrations/plugin.mdx:117]:
```suggestion
This integration adds source file context to stack traces for captured errors from the inline JavaScript inside the page's HTML (e.g. JS in `<script>` tags).

```

File: src/includes/platforms/configuration/integrations/plugin.mdx [hunk 3]
```diff
@@ -107,3 +107,21 @@ Available options:
...
+Note that this integration _cannot_ collect source code from assets referenced by your HTML (e.g. `<script src="..." />`).
```
Comment (shanamatthews) [inline src/includes/platforms/configuration/integrations/plugin.mdx:118]:
```suggestion
This integration _can't_ collect source code from assets referenced by your HTML (e.g. `<script src="..." />`).
```

File: src/includes/platforms/configuration/integrations/plugin.mdx [hunk 4]
```diff
@@ -107,3 +107,21 @@ Available options:
...
+
+Generally, the Sentry backend tries to attach source code to these errors by fetching the page and extracting the respective lines.
```
Comment (shanamatthews) [inline src/includes/platforms/configuration/integrations/plugin.mdx:120]:
```suggestion
This integration is useful when you have inline JS code in HTML pages that can't be accessed by Sentry's backend, for example, due to a login-protected page.

If Sentry can fetch the relevant JavaScript file this integration isn't needed. 
```

File: src/includes/platforms/configuration/integrations/plugin.mdx [hunk 5]
```diff
@@ -107,3 +107,21 @@ Available options:
...
+However, in some cases, our backend might not be able to fetch the JavaScript file, for example if the file is only available for authenticated users.
```
Comment (shanamatthews) [inline src/includes/platforms/configuration/integrations/plugin.mdx:121]:
```suggestion
```

File: src/includes/platforms/configuration/integrations/plugin.mdx [hunk 6]
```diff
@@ -107,3 +107,21 @@ Available options:
...
+
+Use this integration if you have inline JS code in HTML pages that can't be accessed by our backend, for example due to a login-protected page.
```
Comment (shanamatthews) [inline src/includes/platforms/configuration/integrations/plugin.mdx:123]:
```suggestion
```

move up

File: src/includes/platforms/configuration/integrations/plugin.mdx (hunk 4 above)
Comment (shanamatthews) [reply src/includes/platforms/configuration/integrations/plugin.mdx:120]: I tried to combine the sentences here to be more concise and user-focused. Feel free to tweak if I have any details wrong.

**Review by shanamatthews — APPROVED** [review]

File: src/includes/platforms/configuration/integrations/plugin.mdx (hunk 2 above)
Comment (Lms24) [reply src/includes/platforms/configuration/integrations/plugin.mdx]: I tweaked this a little further but thanks for the suggestion!

**Review by shanamatthews — APPROVED** [review]
Looks great now 👍

File: src/platform-includes/configuration/contextlines/javascript.mdx [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+  integrity="sha384-{{@inject packages.checksum('sentry.javascript.browser', 'contextlines.min.js', 'sha384-base64') }}"
+  crossorigin="anonymous"
+></script>
+
+<script>
+  Sentry.onLoad(function () {
+    Sentry.init({
+      integrations: [
+        new Sentry.Integrations.ContextLines({
+          // The number of lines to collect around each stack frame's line number
```
Comment (Lms24) [inline src/platform-includes/configuration/contextlines/javascript.mdx:35]:
```suggestion
          // The number of lines to collect before and after each stack frame's line number
```

**Pull request merged**